homeassistant/components/zha/sensor.py

```diff
@@ -238,6 +238,20 @@ async def _refresh(self, time):
             )
 
 
+# pylint: disable-next=hass-invalid-inheritance # needs fixing
+class EnumSensor(Sensor):
+    """Sensor with value from enum."""
+
+    _attr_device_class: SensorDeviceClass = SensorDeviceClass.ENUM
+    _enum: type[enum.Enum]
+
+    def formatter(self, value: int) -> str | None:
+        """Use name of enum."""
+        assert self._enum is not None
+
+        return self._enum(value).name
```

Comment:
minor, but you wanna remove the empty line in between here?
(I don't think it helps readability)

Reply:
Done